package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "vsts-task-tool-lib",
+  "name": "azure-pipelines-tool-lib",
   "version": "0.10.1",
```

Comment:
major version bump? or does it not matter?

if npm redirects then it will matter (module name changed), otherwise I guess it doesn't matter